src/docs/single_gpu.md

```diff
@@ -35,14 +35,14 @@ The args used in the command above are:
 
 ## How to run with different datasets?
 
-Currently 4 datasets are supported that can be found in [Datasets config file](../llama_recipes/configs/datasets.py).
+Currently 4 datasets are supported that can be found in [Datasets config file](../llama_cookbook/configs/datasets.py).
```

Comment:
Is this link working? Which file are we referring to here? This one? src/llama_recipes/configs/datasets.py